SUPPORT.md

```diff
@@ -6,7 +6,7 @@
 
 This project uses GitHub Issues to track bugs and feature requests. Please search the existing issues before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new issue.
 
-Issues can be created and searched through for existing [issues here](https://github.com/Azure/terraform-azurerm-avm-res-resources-resourcegroup/issues).
+Issues can be created and searched through for existing [issues here](https://github.com/Azure/terraform-azurerm-avm-res-resources-resourcgroup/issues).
```

Comment:
This isn't correct, it's set by the grept tool (if the env vars are set) or just can fix manually

```suggestion
Issues can be created and searched through for existing [issues here](https://github.com/Azure/terraform-azurerm-avm-res-resources-resourcegroup/issues).
```